runtime.md

```diff
@@ -122,8 +137,3 @@ This specification does not mandate the name of this JSON file.
 See the specification of the `config.json` file for the definition of these fields.
 The stopping, or exiting, of these secondary process MUST have no effect on the state of the container.
 In other words, a container (and its PID 1 process) MUST NOT be stopped due to the exiting of a secondary process.
-
-## Hooks
-
-Many of the operations specified in this specification have "hooks" that allow for additional actions to be taken before or after each operation.
```

Comment:
Yeah. That was what I was suggesting during the weekly meetings.

On Fri, Apr 22, 2016 at 11:10 AM, Doug Davis <notifications@github.com>
wrote:

> In runtime.md
> <https://github.com/opencontainers/runtime-spec/pull/384#discussion_r60780090>
> :
>
> > @@ -122,8 +137,3 @@ This specification does not mandate the name of this JSON file.
> >  See the specification of the `config.json` file for the definition of these fields.
> >  The stopping, or exiting, of these secondary process MUST have no effect on the state of the container.
> >  In other words, a container (and its PID 1 process) MUST NOT be stopped due to the exiting of a secondary process.
> > -
> > -## Hooks
> > -
> > -Many of the operations specified in this specification have "hooks" that allow for additional actions to be taken before or after each operation.
>
> It seems to me then we should split stop/delete at the same time, no?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/opencontainers/runtime-spec/pull/384/files/5813d96a74b5e753851fdad11ce4731ac7be0576#r60780090>
>